### ISSUE TITLE
How to add the button again, my be on demand of a button click

### ISSUE THREAD

**Issue by thivankasarathchandra** [Smueez/floating_animated_widget#7]
Once we delete the button, is there another way to reopen the button ?

**Comment by Smueez** [comment]
nope, not at this moment. plz, feel free to share if you have any ideas on how to implement this, or maybe u can create a PR, would be much appreciated. thank u
